suzieq/db/parquet/parquetdb.py

```diff
@@ -224,13 +224,13 @@ def write(self, table_name: str, data_format: str,
             if filename_cb:
                 pq.write_to_dataset(table, root_path=folder,
                                     partition_cols=partition_cols,
-                                    version="2.0", compression="ZSTD",
+                                    version="2.4", compression="ZSTD",
```

Comment:
Can I be Lorina and say why not use a #define constant to hold the parquet version :)